fix_public.py

```
@@ -0,0 +1,29 @@
+    "src/main/java/io/github/carlos_emr/carlos/managers/PreventionManager.java"
+]
+
+for fpath in files:
+    with open(fpath, "r") as f:
+        lines = f.readlines()
+    
+    out_lines = []
+    for line in lines:
+        if line.startswith("    ") and not line.startswith("    public ") and not line.startswith("    //") and not line.startswith("    /*") and not line.startswith("    *"):
+            # Check if it looks like a method signature
+            if re.match(r'^    [\w<>,\[\] ]+\s+\w+\(.*', line):
```

Ruff (0.15.12)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)